include/ddc/discrete_domain.hpp

```diff
@@ -427,6 +427,16 @@ KOKKOS_FUNCTION constexpr auto remove_dims_of(
     return detail::convert_type_seq_to_discrete_domain<type_seq_r>(DDom_a);
 }
 
+// Remove the dimensions DDimsB from DDom_a
+template <class... DDimsB, class DDomA>
+KOKKOS_FUNCTION constexpr auto remove_dims_of(DDomA const& DDom_a) noexcept
+{
+    return remove_dims_of(
+            DDom_a,
+            ddc::DiscreteDomain<DDimsB...>(
+                    ddc::DiscreteElement<DDimsB...>(ddc::DiscreteElement<DDimsB>(0)...),
+                    ddc::DiscreteVector<DDimsB...>(ddc::DiscreteVector<DDimsB>(0)...)));
```

Comment:
I would prefer to avoid creating raw `DiscreteElement` out of integers. Can you copy the implementation of the other `remove_dims_of`.